modules/op-release-notes-1-10.adoc

```diff
@@ -166,3 +166,13 @@ With this update, {pipelines-title} General Availability (GA) 1.10.1 is availabl
 * Before this update, when a `PipelineRun` object could not be created, the error received from the Tekton controller was not reported to the user. With this update, {pac} reports the error messages to the GitHub interface so that users can troubleshoot the errors. {pac} also reports the errors that occurred during pipeline execution.
 * With this update, {pac} does not echo a secret to the GitHub checks interface when it failed to create the secret on the {product-title} cluster because of an infrastructure issue.
 * This update removes the deprecated APIs that are no longer in use from {pipelines-title}.
+
+[id="release-notes-1-10-2_{context}"]
+== Release notes for {pipelines-title} General Availability 1.10.2
+
+With this update, {pipelines-title} General Availability (GA) 1.10.2 is available on {product-title} 4.11 and later versions.
+
+[id="fixed-issues-1-10-2_{context}"]
+=== Fixed issues
+
+Before this update, Tekton Operator would not support the `enable-api-fields` flag in the `TektonConfig` CR, which prevented the deployment of the service. With this update, Tekton Operator supports these values, `alpha`, `beta`, and `stable` for the `enable-api-fields` flag in the `TektonConfig` CR.
```

Comment:
`enable-api-fields` was supported in previous versions as well. The issue was we could set it only to `alpha` or `stable`. With this fix we can add `beta` value as well.

Reply:
We support all these(`alpha`, `beta`, and `stable`) values in previous version as well. Due to a bug in product, user can not set `beta`. We have fixed that issue in this release.